torch_xla/core/xla_model.py

```diff
@@ -92,6 +131,12 @@ def xla_real_devices(devices):
 
 
 def xla_replication_devices(local_devices):
+  """
```

Comment:
I would not document this, especially with these very simple comments.
Leave them out as those are internal.

Reply:
_xla_replication_devices?

Reply:
Not that private :smile: ... we use it internally.
I'd leave the name w/out '_' as usually by our convention, that is file private.